README.md

```diff
@@ -2,7 +2,11 @@
 
 [![npm version](https://badge.fury.io/js/rendition.svg)](http://badge.fury.io/js/rendition)
 [![Build Status](https://travis-ci.org/resin-io-modules/rendition.svg?branch=master)](https://travis-ci.org/resin-io-modules/rendition)
+[![Dependency Status](https://img.shields.io/david/resin-io-modules/rendition.svg)](https://david-dm.org/resin-io-modules/rendition)
+[![npm](https://img.shields.io/npm/dm/rendition.svg?maxAge=2592000)](https://www.npmjs.com/package/rendition)
+[![GitHub last commit](https://img.shields.io/github/last-commit/resin-io-modules/rendition.svg)](https://github.com/resin-io-modules/rendition/commits/master)
 [![style: styled-components](https://img.shields.io/badge/style-%F0%9F%92%85%20styled--components-orange.svg?colorB=daa357&colorA=db748e)](https://github.com/styled-components/styled-components)
+[![Join the chat at https://gitter.im/resin-io-modules/rendition](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/resin-io-modules/rendition?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
This gitter chat link doesn't work for me, I just get a 404. I think nowadays we mostly avoid having per-project gitters like this, since they're a hassle to keep an eye on and to maintain, and we push people towards the general resin one, or the forums.

Reply:
Moving forwards rendition is going to be pushed as a separate entity/brand to resin, similar to etcher, which is why we it has its own gitter chatroom. I'm not sure why its getting a 404

Reply:
Yep, I understand we're going to push it as it's own open-source project, but is gitter definitely our preferred route for user communication with our open source projects? My impression was that in general we were moving away from it everywhere we can, not setting up new ones.

Fine if we've discussed this already somewhere, but might be worth a #process around this if not. Could set up a Rendition section in the forum instead, for example, or... something else. Or maybe it's fine, I just don't know.

Reply:
I'll bring it up via #process

Reply:
For now I'm going to remove the gitter badge for now so we can move forward with this PR